### PR TITLE
add convenience methods to support easy response validation

### PR DESCRIPTION
**Pull request by aburmeis**
See #156

### CODE REVIEW

**Comment by aburmeis** [comment]
I tried to use your code style but it seems at least I missed the import order, sorry.

File: openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/validation/RequestValidator.java [hunk 1]
```diff
@@ -107,8 +129,33 @@ public OperationValidator getValidator(final Path path, final Operation operatio
    * @param request The request to validate. Must be {@code nonnull}.
    * @throws ValidationException A validation report containing validation errors
    */
-  public RequestParameters validate(final Request request) throws ValidationException {
-    return validate(request, new ValidationData<>());
+  public void validate(final Request request) throws ValidationException {
```
Comment (llfbandit) [inline openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/validation/RequestValidator.java:132]: Why did you remove the returned parameters?

**Review by llfbandit — CHANGES_REQUESTED** [review]

File: openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/validation/RequestValidator.java (hunk 1 above)
Comment (aburmeis) [reply openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/validation/RequestValidator.java:132]: Sorry, that was a mistake, will revert
